src/handlers/uniswap.ts

```diff
@@ -105,7 +109,7 @@ export const getStateDiff = async (
       value: '0x0',
     },
     ['stateDiff'],
-    'latest',
+    blockNumber - Number(config.trace_call_block_number_offset ?? 20),
```

Comment:
oh maybe this will fail if the app is not getting config.trace_call_block_number_offset, `'' ?? 20` is `''` i think we should use `'' || 20`

Reply:
Makes sense! 

```suggestion
    blockNumber - Number(config.trace_call_block_number_offset || 20),
```